sematic/api_client.py

```diff
@@ -536,6 +537,25 @@ def update_run_future_states(run_ids: List[str]) -> Dict[str, FutureState]:
     return result_dict
 
 
+def save_metric_points(metric_points: List[MetricPoint]) -> None:
```

Comment:
Please consider using `*args` instead of a list, for syntactic sugar.

Reply:
This is an internal API, does not really need sweetening.